templates/scenarios/js/notification-http-timer-trigger/teamsfx/app.yml

```diff
@@ -0,0 +1,73 @@
+version: 1.0.0
+
+provision:
+  - uses: botAadApp/create # Creates a new AAD app for Bot Registration.
+    with:
+      name: notification-bot
+    # Output: following environment variable will be persisted in current environment's .env file.
+    # BOT_ID: the AAD app client id created for Bot Registration.
+    # SECRET_BOT_PASSWORD: the AAD app client secret created for Bot Registration.
+
+  - uses: arm/deploy # Deploy given ARM templates parallelly.
+    with:
+      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}} # The AZURE_SUBSCRIPTION_ID is a built-in environment variable. TeamsFx will ask you select one subscription if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select subscription if it's empty in this case.
+      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}} # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable. TeamsFx will ask you to select or create one resource group if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select or create resource grouop if it's empty in this case.
+      templates:
+      - path: ./infra/azure.bicep
+        parameters: ./infra/azure.parameters.json
+        deploymentName: Create-resources-for-bot
+      bicepCliVersion: v0.9.1 # Teams Toolkit will download this bicep CLI version from github for you, will use bicep CLI in PATH if you remove this config.
+    # Output: every bicep output will be persisted in current environment's .env file with certain naming conversion. Refer https://aka.ms/teamsfx-provision-arm#output for more details on the naming conversion rule.
+
+deploy:
+  - uses: npm/command
+    with:
+      workingDirectory: ./bot
+      args: install
+  - uses: npm/command
+    with:
+      workingDirectory: ./bot
+      args: run build
+  - uses: azureFunctions/deploy
+    with:
+      workingDirectory: ./bot
+      # deploy base folder
+      distributionPath: ./
+      # can be changed to any ignore file location, leave blank will ignore nothing
+      ignoreFile: ./.funcignore
+      # the resource id of the cloud resource to be deployed to
+      resourceId: ${{BOT_AZURE_FUNCTION_APP_RESOURCE_ID}}
```

Comment:
we should have a .appserviceIgnore file for deploy and a parameter for it here.
please sync with @Siglud

Reply:
http-timer-trigger does not have any app service resources?

Reply:
If we deploy to Azure Functions, then we use .funcignore instead of .appserviceIgnore